www/viewgame

```diff
@@ -1310,13 +1307,43 @@ if (count($links) == 0 && $dlnotes == "") {
                 . "class=\"zip-contents-arrow\">Contains "
                 . "<b>$compPri</b></span><br>";
         }
-
+       
         if ($linkdesc) {
             $linkdesc = preg_replace(
                 "/([a-z0-9][\/.])([a-z0-9])/i", "$1$zwsp$2", $linkdesc);
             echo "<span class=dlnotes>$linkdesc<br></span>";
         }
 
+        // Don't show "Play Online" mini link for HTML links; that's what the link itself does
+        //   ... unless the HTML is zipped, in which case we *do* need a Play Online link
+        $playOnlineMiniLink = "";
+        if ($playOnlineURL && ($onlineInterpreterType !== "HTML" || $compPri)) {
+            $playOnlineMiniLink = "<div class='details'><a href='$playOnlineURL'>Play Online</a></div>";
+            echo $playOnlineMiniLink;
+        }
+
+        // If we're linking to a game that needs an interpreter, expand the format 
+        // description to include instructions for downloading and running it.
+        if ($fmtclass == "G" && $fmtexternid != "tads3web" && $fmtexternid != "hypertextgame") {
+            $format_instructions = "";
+            if ($playOnlineMiniLink) {
+                // These instructions are following a "play online" mini link, so we can begin with "Or"
+                $format_instructions = "Or " ;
+            } else {
+                $format_instructions = "To play, you can ";
+            }
+            $format_instructions .= 'download <a href=\"$url\">' . $linktitle . '</a>';
```

Comment:
```suggestion
            $format_instructions .= "download <a href=\"$url\">$linktitle</a>";
```

Reply:
@dfabulich I made this change. Should I also go back to my other double quoted strings and include the variables inside instead of concatenating them?